src/timedata.cpp

```diff
@@ -3,16 +3,14 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include "timedata.h"
```

Comment:
lets keep `timedata.h` include at the top.

also, need to include `chainparams.h` to pickup `Params()` and appease CMake